docker-compose.yml

```diff
@@ -64,7 +64,23 @@ services:
   hapi-fhir:
     image: jembi/hapi:v7.0.3-wget
     ports:
-      - 8081:8080
+      - 3447:8080
+    networks:
+      - smart-health-links-portal-network
+    volumes:
+      - hapi_fhir_data:/var/lib/hapi-fhir
+    environment:
+      - hapi.fhir.allow_external_references=true
+      - hapi.fhir.bulk_export_enabled=true
+      - HAPI_FHIR_IPS_ENABLED=true
```

Comment:
Just confirm if this env is valid? the format seems a bit different to the others. This might be set as an env, but not actually applied in the FHIR